### PR TITLE
sent client side timestamp with every event

### PR DESCRIPTION
**Pull request by fippo**
I want to look into client side timing so doing things after the transport (and maybe GC on the server) ...

### CODE REVIEW

File: rtcstats.js [hunk 1]
```diff
@@ -30,6 +30,7 @@
   function trace() {
     //console.log.apply(console, arguments);
     // TODO: drop getStats when not connected?
+    arguments.push(new Date().getTime());
```
Comment (juandebravo) [inline rtcstats.js]: @fippo `arguments` is not an array object. Please [avoid mutating it](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments)

``` js
var args = Array.prototype.slice.call(arguments);
args.push(new Date().getTime());
```

File: rtcstats.js [hunk 2]
```diff
@@ -30,10 +30,12 @@
   function trace() {
     //console.log.apply(console, arguments);
     // TODO: drop getStats when not connected?
+    var args = Array.prototype.slice.call(arguments);
+    args.push(new Date().getTime());
```
Comment (juandebravo) [inline rtcstats.js:34]: thanks @fippo!

WDYT about sending the ISO format?

``` js
args.push((new Date()).toISOString());
```

Comment (fippo) [inline rtcstats.js:34]: timestamps are more compact for transmission

Comment (juandebravo) [inline rtcstats.js:34]: ack, thanks!

File: rtcstats.js (hunk 1 above)
Comment (fippo) [inline rtcstats.js]: argh. the server side assumes this to be an object right now, in particular when setting the server side timesttamp :-(

**Pull request merged**